server/players.py

```diff
@@ -35,18 +35,21 @@ def __init__(
         permission_group: int = 0,
         lobby_connection: "LobbyConnection" = None
     ):
-        super().__init__(player_id, login)
+        self._faction = 0
```

Comment:
typing for local variables + typing for incoming variables like `clan`, `global_rating`, `ladder_rating`.

Reply:
I had no reason to touch this code beyond copying it. Faction could use an enum anyway rather than just an obvious typing hint.

Reply:
You can make it better ;-) you're refactoring, isn't it?

Reply:
Whenever possible, one commit is one feature.